etc/examples/argocd-app.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: rhoc
+spec:
+  destination:
+    namespace: cos
+    server: https://kubernetes.default.svc
+  project: default
+  source:
+    path: etc/kubernetes/manifests/overlays/lburgazzoli
```

Comment:
is this a leftover?

Reply:
yes, it will be removed before marking the PR as ready